deploy/docker/hail_search/sync_datasets

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+suffixes=(
+    GRCh37_SNV_INDEL_VERSION
+    GRCh37_MITO_VERSION
+    GRCh37_SV_VERSION
+    GRCh37_GCNV_VERSION
```

Comment:
we are not going to have GRCh37 SV, GCNV, or MITO data (we will never have internal data for it, so it would only be possible once we make these available to anvil users, which will involve lots of code changes) so we may want to take those out

Reply:
Yep, I think just removing for now makes sense.